collections/cnes-land-cover-map.yaml

```diff
@@ -95,7 +95,7 @@ Extent:
     interval:
```

Comment:
Putting it here as I can't put it at place. Please also add 2021 to line 85 "... - Legend used for 2020, 2019 and 2018 data"

Reply:
Missed it, changed now :)